tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "include": ["src"],
+  "compilerOptions": {
+    "target": "ESNext",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "module": "Node16",                                  /* Specify what module code is generated. */
```

Comment:
I started replacing Node16 with NodeNext everywhere. Apparently it's more future proof

Reply:
Is there a tsconfig that I can safely copy and use here? Fragment Plugin?

Reply:
[Swup's tsconfig](https://github.com/swup/swup/blob/master/tsconfig.json) is the one I would use for new TypeScript projects.

Reply:
Copied the fragment plugin tsconfig :)

Reply:
ok :)

Reply:
Just checked and it looks like they're identical.